packages/rehype-shiki/src/languages.mjs

```diff
@@ -33,6 +37,10 @@ export const LANGUAGES = [
   ...dockerLanguage,
   ...diffLanguage,
   ...yamlLanguage,
+  ...cLanguage,
```

Comment:
FYI this increases the bundle size on the website (client-side) because these are all imported and exposed.

IMO, we could make a base set of languages, and simply concate the extra ones on api-docs-tooling? I'm also fine with this.

Reply:
You know 600kb is awful lot for a client to download. This would painfully increase the load times.

Reply:
Yup, that is correct.